doc/man3/OSSL_CRMF_MSG_set0_validity.pod

```diff
@@ -37,29 +38,30 @@ OSSL_CRMF_MSGS_verify_popo
 
 =head1 DESCRIPTION
 
-OSSL_CRMF_MSG_set_validity() sets B<from> as notBefore and B<to> as notAfter
-as the validity in the certTemplate of B<crm>.
+OSSL_CRMF_MSG_set0_validity() sets the I<notBefore> and I<notAfter> fields,
+as far as the respective argument is not NULL,
+as validity constraints in the certTemplate of I<crm>.
```

Comment:
The parameters are always set, whether the parameter is NULL or not. Maybe you should instead say what setting it to NULL means.

The "as far as" also sounds weird to me, "when" is probably easier.

Reply:
I've re-phrased this.